templates/prescription/general_edit.html

```diff
@@ -120,8 +120,8 @@
 </table>
 
 {if $GLOBALS.enable_amc_prompting}
-  <div style='float:right; margin-right: 25px; border-style: solid; border-width: 1px;'>
-    <div style='float:left; margin:5px 5px 5px 5px;'>
+  <div class='float-right mr-25' style='border-style: solid; border-width: 1px;'>
```

Comment:
No such thing as `mr-25` -- only goes up to 5.

Reply:
@tywrenn sad life :(